meetings/2019-08-28.md

```diff
@@ -0,0 +1,103 @@
+    * Of note, date for next years Node+JS Interactive (2020) may shift. Possibly late spring/early
+      summer, October also might be considered. Event space availability is one constraint.
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* stream: don't emit error after destroy() [#29197](https://github.com/nodejs/node/pull/29197)
+  * Matteo, significant change for behaviour in streams. When streams errors, still possible
+    in some cases error may still be emitted again.  Discussion is around SemVer major change
+    if Destroy is called a second time, error would not be emitted.
+  * After destroy, the call with a second time with error.  
+  * Seems like we are at a stalemate at this point.
+  * Gabriel: how useful to get error from second Destroy.  Seems like semantics of Destroy 
+    would be to release all resources, so getting an error on second Destroy seems like its just
+    a JavaScript side effect, not something you.d expect.
+  * Matteo, if it was me, if you do anything after Destroy, generate/return error saying already 
+    destroyed/closed.  Concern is that we have support the behaviour and potential breakage.
+  * Jemeriah, if you call send time should we just though instead. Matteo fears that may be more
+    breaking.  
+  * Matteo, lets explore the option of throw if you call after Destroyed/Closed
+  * Next action: Jeremiah will write up his suggestion on Issue and we.ll go from there.
+
+* buffer: add Buffer.harden() method [#28439](https://github.com/nodejs/node/pull/28439)
+  * Next action: please review and comment or approve.
+
+* build: ongoing list of actions for Python 3 compatibility [#25789](https://github.com/nodejs/node/issues/25789)
+  * covered under strategic initiatives discussion 
+
+* http2: make compat finished match http/1 [#24347](https://github.com/nodejs/node/pull/24347)
+  * Original exception from Anatoli
+  * Matteo: Some engagement but not too much
+  * Matteo, Anna/Jeremiah can you take a look.
+  * Matteo is fine with it.
+  * Will wait for input from Anna/Jeremiah and if they think its ok and we don.t have any more
+    input/engagement from Anatoli the objection will be over ridden based on our governance.
+
+### nodejs/TSC
+
+* Nominating @tniessen for the TSC [#746](https://github.com/nodejs/TSC/issues/746)
+* Nominating @BethGriggs to the TSC [#718](https://github.com/nodejs/TSC/issues/718)
+
+## Strategic Initiatives
+
+* N-API 
+  * Working on worshop for Node+JS interactive
+
+* Core promise APIs
+  * Have been slowly progressing to add to Event Emitter, getting back to have more time on this
+  * Also working on proposal for Emitter, ask if people can look at PR
+    https://github.com/nodejs/node/pull/27867
+ 
+* New Stream APIs 
+  * Nothing new to mention this time
+
+* Python3 
+  * Claus has flagged that  https://github.com/nodejs/node/issues/29246 needs help
```

Comment:
FTR, I tracked this problem down to npm's copy of node-gyp only working with py2: https://github.com/nodejs/node/issues/29246#issuecomment-528989120